tests/Resources/ti.require.addontest.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* global Ti */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions');
+
+describe('require()', function () {
+	it('JSON-based require() with single-quotes', function () {
+		var amber = require('json_files/amber');
```

Comment:
Don't use a bad require style 😏 
```javascript
var amber = require('./json_files/amber');
```

Reply:
Updating now. I was not sure if changes to the the `tests` repository will be picked up during CI build, so I placed it in the main repo instead. Good to know!